mmdet/core/fp16/hooks.py

```diff
@@ -127,7 +127,8 @@ def patch_norm_fp32(module):
     """
     if isinstance(module, (nn.modules.batchnorm._BatchNorm, nn.GroupNorm)):
         module.float()
-        if isinstance(module, nn.GroupNorm) or torch.__version__ < '1.3':
+        if (isinstance(module, nn.GroupNorm) or torch.__version__ < '1.3'
```

Comment:
The condition `torch.__version__ < '1.3'` can be removed since we only support pytorch 1.3+.